README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# hw01
+
+![](https://github.com/jchon11/myrepo/blob/master/joshchon.png)
+## Joshua Chon
+
+### Summary
+My name is Joshua Chon. I am a **MAPSS Psychology Student** currently attending my third quarter at the *University of Chicago*. Currently, I work in the **APEX Lab**.
+
+### Research Interests
+My research interests lie in cognitive psychology, particularly in *perception*, *language* and *sleep*. My current project deals with **auditory perception** and how it relates to different individuals. 
+
+### Hobbies
+My hobbies include:
+1. Hiking
+2. Camping
+3. Biking
+4. Watching movies. My favorite movies are...
+  * *The Godfather*
+  * *Citizen Kane*
+  * *The Grand Budapest Hotel*
+  
+### Link
+https://apexlab.uchicago.edu/
```

Comment:
To link inline use `[TEXT](link.com)`. (Inline means it renders [like this](https://apexlab.uchicago.edu/))